docs/visualization/visualize_unit_responses.ipynb

```diff
@@ -83,7 +83,7 @@
      "name": "stderr",
```

Comment:
<p>Would it be possible to choose a time with a clear sensory response in the image at time zero? How did you choose this time? </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AllenInstitute/openscope_databook/pull/133/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>